deploy/olm-catalog/qdrouterd-operator/0.1.0/qdrouterd-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -68,12 +78,12 @@ spec:
           selector:
             matchLabels:
               name: qdrouterd-operator
+          strategy: {}
```

Comment:
Is empty struct required?

Reply:
changes made to fix those unnecessary write spaces and position movement.